docs/known-issues.md

```diff
@@ -12,16 +12,186 @@ The workaround is to edit the migration code generated by EF Core to make it wor
+
+By default, EF Core will generate a set of SQL commands to 
+drop the PK constraint
+rename the table, and
+re-add the PK constraint
+
+The workaround is to edit the migration code generated by EF Core to make it work with CockroachDB.
+
+### CockroachDB doesn't support function that returns TABLE
+Github issue: https://github.com/cockroachdb/cockroach/issues/100226 
+
+### CockroachDB doesn't support make_date, make_timestamp, make_timestamptz
+Github issue: https://github.com/cockroachdb/cockroach/issues/108448 
+
+These functions are supported in latest 23.1 build
+
+### CockroachDB doesn't support CREATE COLLATION, DROP COLLATION
+Github issue: https://github.com/cockroachdb/cockroach/issues/59567
+
+This is a known limitation of CockroachDB. EFCore supports for creating collation will not work.
+
+### CockroachDB doesn’t support ADD GENERATED, SET GENERATED in ALTER COLUMN
+Github issue: https://github.com/cockroachdb/cockroach/issues/110010 
+
+This impacts code first migration when the following extension methods are used to add or set an existing column as identity column
+
+UseIdentityAlwaysColumn
+UseIdentityByDefaultColumn
+
+### CockroachDB doesn't support creating indexes with operators
+Github issue: https://github.com/cockroachdb/cockroach/issues/110247
+
+This impacts code first migration when the extension method HasOperator is used to create index
+
+For example:
+```csharp
+builder => builder.Entity("People").HasIndex("FirstName", "LastName")
+                .HasOperators("text_pattern_ops")
+```
+
+### CockroachDB doesn’t support RESTART WITH, SET INCREMENT BY in ALTER COLUMN
+Github issue: https://github.com/cockroachdb/cockroach/issues/110015 
+
+This impacts code first migration when the extension method HasIdentityOptions is used to update the options for an identity column 
+
+For example:
+
+modelBuilder.Entity<Blog>().Property(b => b.Id).HasIdentityOptions(startValue: 100, incrementBy: 2)
+
+### CockroachDB doesn’t support CYCLE option for sequences
+Github issue: https://github.com/cockroachdb/cockroach/issues/20961 
+
+In PostgreSQL, The CYCLE option allows the sequence to wrap around when the maxvalue or minvalue has been reached by an ascending or descending sequence respectively. If the limit is reached, the next number generated will be the minvalue or maxvalue, respectively.
+
+However, CockroachDB doesn’t support this option. Code that uses the extension method HasIdentityOptions to make a sequence cyclic will not work
+
+Example:
+```
+builder => builder.Entity("People").Property<int?>("SomeColumn")
+                .UseIdentityByDefaultColumn()
+                .HasIdentityOptions(
+                    startValue: 5,
+                    incrementBy: 2,
+                    minValue: 3,
+                    maxValue: 2000,
+                    cyclic: true,
+                    numbersToCache: 10)
+```
+
+### CockroachDB doesn't truly support infinity
```

Comment:
"doesn't truly support infinity" is odd phrasing, esp since we do for decimals and floats

i'd say "doesn't support `infinity` for timestamp and date types"